packages/fxa-settings/src/models/Account.ts

```diff
@@ -1266,7 +1270,7 @@ export class Account implements AccountData {
       { kB: opts.kB },
       { sessionToken: true, keys: true }
     );
-    currentAccount(currentAccount(getOldSettingsData(data)));
+    currentAccount(currentAccount(getStoredAccountData(data)));
```

Comment:
I am not sure why we need a double `currentAccount` here but I know it's how it was previously so that's neither here nor there for this PR.